sms_twilio.js

```diff
@@ -14,6 +14,38 @@ sms.configure = function (options) {
   sms.from = options.from;
 };
 
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+/**
+ * Lookup phone number information through twilio lookups api.
+ * @param phone
+ */
+sms.lookup = function (phone) {
+  var lookup = lookups.findOne({ phone_number: { $regex: phone }});
```

Comment:
So the regex match is to get around having to enter +1. 
555 555555 matches +1 555 5555555.